app/src/main/java/org/fossasia/openevent/api/processor/SpeakerListResponseProcessor.java

```diff
@@ -57,6 +57,6 @@ public void run() {
     @Override
     public void failure(RetrofitError error) {
         // Do something with failure, raise an event etc.
-        OpenEventApp.getEventBus().post(new SponsorDownloadEvent(false));
+        OpenEventApp.getEventBus().post(new SpeakerDownloadEvent(false));
```

Comment:
Yeah this isn't related to showing up of dialogs, this is possibly a small typo @mananwason did, so I fixed it too.